committing.rst

```diff
@@ -235,10 +251,10 @@ it is common to simply paste the NEWS entry into the commit message.  Here
 is an example::
 
    bpo-42: the spam module is now more spammy.
-   
+
    The spam module sporadically came up short on spam. This change
    raises the amount of spam in the module by making it more spammy.
-   
+
```

Comment:
I have never added spaces before acknowledgements before and I believe same was generally true.  Unless the resulting News/changelog is formatted differently from before, blank lines might make it harder, not easier to read.